content/docs/faq-styling.md

```diff
@@ -1,51 +1,51 @@
 ---
 id: faq-styling
-title: Styling and CSS
+title: Ձևավորում և CSS
 permalink: docs/faq-styling.html
 layout: docs
 category: FAQ
 ---
 
-### How do I add CSS classes to components? {#how-do-i-add-css-classes-to-components}
+### Ինչպե՞ս կարող եմ կոմպոնենտներին ավելացնել CSS կլասներ {#how-do-i-add-css-classes-to-components}
 
-Pass a string as the `className` prop:
+Փոխանցել տողշ\` որպես `className`-ի ատրիբուտ\`
 
 ```jsx
 render() {
-  return <span className="menu navigation-menu">Menu</span>
+  return <span className="menu navigation-menu">Մենյու</span>
 }
 ```
 
-It is common for CSS classes to depend on the component props or state:
+Սովորաբար CSS կլասները կախված են կոմպոնենտի props-ից կամ state-ից\`
 
 ```jsx
 render() {
   let className = 'menu';
   if (this.props.isActive) {
     className += ' menu-active';
   }
-  return <span className={className}>Menu</span>
+  return <span className={className}>Մենյու</span>
 }
 ```
 
->Tip
+>Խորհուրդ
 >
->If you often find yourself writing code like this, [classnames](https://www.npmjs.com/package/classnames#usage-with-reactjs) package can simplify it.
+>Եթե դուք հաճախ եք գրում նմանատիպ կոդ, [classnames](https://www.npmjs.com/package/classnames#usage-with-reactjs)<sub>eng</sub> հավելումը ավելի հեշտ կդարձնի այն։
 
-### Can I use inline styles? {#can-i-use-inline-styles}
+### Կարո՞ղ եմ օգտագործել ներկառուցված ձևավորումներ {#can-i-use-inline-styles}
 
-Yes, see the docs on styling [here](/docs/dom-elements.html#style).
+Այո, տես ձևավորման փաստաթղթավորումն [այտեղ](/docs/dom-elements.html#style)։
 
-### Are inline styles bad? {#are-inline-styles-bad}
+### Արդյո՞ք ներկառուցված ձևավորումները վատ են {#are-inline-styles-bad}
 
-CSS classes are generally better for performance than inline styles.
+CSS կլասներն ընդհանուր առմամբ ավելի լավ են արտադրողականության համար, քան\` ներկառուցված ձևավորումները։
 
-### What is CSS-in-JS? {#what-is-css-in-js}
+### Ի՞նչ է CSS-in-JS-ը {#what-is-css-in-js}
 
-"CSS-in-JS" refers to a pattern where CSS is composed using JavaScript instead of defined in external files. Read a comparison of CSS-in-JS libraries [here](https://github.com/MicheleBertoli/css-in-js).
+«CSS-in-JS»-ը օրինակ է, որտեղ CSS-ը արտաքին ֆայլերում գրելու փոխարեն այն ստեղծվում է JavaScript-ի միջոցով։ Կարդա CSS-in-JS գրադարանների համեմատությունն [այստեղ](https://github.com/MicheleBertoli/css-in-js)<sub>eng</sub>։
```

Comment:
«CSS-in-JS»-ը ձևանմուշի հղում է, որտեղ...

Reply:
"արտաքին ֆայլերում գրելու փոխարեն" ֊> "արտաքին ֆայլերում սահմանվելու փոխարեն"

Reply:
> «CSS-in-JS»-ը ձևանմուշի հղում է, որտեղ...

սխալ եմ գրել, «հղվում է ձևանմուշին»